mods/default/nodes.lua

```diff
@@ -151,6 +151,10 @@ default:sign_wall
 default:ladder
 
 default:fence_wood
+default:fence_acacia_wood
+default:fence_junglewood
```

Comment:
Right, but slabs and stairs are also ```junglewood``` and not ```jungle_wood```. Hence Introducing one node that is off seemed confusing.